src/views/__tests__/FileUpload.test.js

```diff
@@ -8,6 +8,24 @@ import {mocks} from '../../../__mocks__/kf-api-study-creator/mocks';
 
 jest.mock('auth0-js');
 
+beforeAll(() => {
```

Comment:
this looks like a duplicate from __setup__, is it necessary here or does the setup logic take care of it?

Reply:
this part is used to target document to render